Makefile

```diff
@@ -20,6 +20,7 @@ LDFLAGS = -ldflags "-X $(VERSION_PKG).VERSION=$(VERSION) -X $(VERSION_PKG).COMMI
 
 DOCKER_RUN_OPTS ?=
 DOCKER_RUN_ARG ?=
+RUN_ARGS ?= -http-addr=localhost:2000
```

Comment:
can we add this directly to the target because we always have to set `RUN_ARGS` when running  locally for setting the teams webhook url.